phpBB/phpbb/db/migration/tool/module.php

```diff
@@ -81,6 +81,9 @@ public function get_name()
 	*/
 	public function exists($class, $parent, $module)
 	{
+		// Clear the Modules Cache first to get actual data
+		$this->cache->destroy("_modules_$class");
```

Comment:
But how it worked before in the ```add()``` method and why tests didn't fail on that, just curious. These things with migrations are all weird 😕